### ISSUE TITLE
'messages' in BOINC-Manager blinking red unfounded

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#862]
**Reported by koyaanisqatsi on 30 Jun 39328963 07:06 UTC**
Sometimes you will get messages in your BOINC-Manager, that are written in red colour. For example "BOINC can't reach Internet". Or "Project xyz can't connect to the Internet". In that case the messages-button in your BOINC-Manager is blinking red.

Now you have to click on messages, then close messages, and the blinking disappears.

From now on, everytime you click on "Advanced view" and choose "View / Normal view" to go back to the BOINC-Manager, the messages-button is blinking red again. So BOINC has "forgotten", that you have already seen the reason for these messages.

In my opinion this is a bug, because it's not pleasant to see something red blinking on my desktop. Therefore I'm forced everytime I want to take a look at my "Advanced view" to confirm this blinking button.

Migrated-From: http://boinc.berkeley.edu/trac/ticket/885

**Comment by romw** [comment]
**Commented by Ageless on 4 Sep 39333445 15:06 UTC**
The messages button in simple view will start blinking every time that a new message is written into the messages window. It isn't restricted to warning messages only. 

If you do not want to see that, use advanced view, or just park BOINC Manager in your system tray.

**Comment by romw** [comment]
**Commented by koyaanisqatsi on 16 Mar 39394784 03:06 UTC**
With respect, but your answer is 100% wrong. I have tried that several times.

When I start BOINC, I will get some "normal" messages. Nothing is blinking red. After the first "red" message, like "no work available" or so, the messages button is blinking red. And every time, I choose "advanced view", this button is blinking red again.

IMHO this is a bug. You argue "If you don't want to see a bad site of BOINC don't choose this bad site". Wow!!! What a weak argumentation.

When I have acknowlegded a message of BOINC, this message may never ever be presented me once again. This is bad behaviour of BOINC.

BOINCs behaviour should be corrected. Not mine :)

Apologies for my bad english.
